genai-perf/genai_perf/main.py

```diff
@@ -193,7 +195,13 @@ def run():
         args.func(args)
     else:
         create_artifacts_dirs(args)
-        tokenizer = get_tokenizer(args.tokenizer)
+        tokenizer = (
+            get_tokenizer(
+                args.tokenizer,
+                args.tokenizer_trust_remote_code,
+                args.tokenizer_revision,
+            ),
+        )
```

Comment:
forgot to add this to the previous review
```suggestion
        tokenizer = get_tokenizer(
            args.tokenizer,
            args.tokenizer_trust_remote_code,
            args.tokenizer_revision,
        )
```

Reply:
this is probably fall out from the formatting.

Reply:
Good catch, fixed!